x/bank/client/cli/cli_test.go

```diff
@@ -180,22 +189,27 @@ func (s *IntegrationTestSuite) TestNewSendTxCmd() {
 	val := s.network.Validators[0]
 	clientCtx := val.ClientCtx.WithOutput(buf)
 
+	ctx := context.Background()
+	ctx = context.WithValue(ctx, client.ClientContextKey, &clientCtx)
+
 	testCases := []struct {
 		name         string
+		from, to     sdk.AccAddress
```

Comment:
It is fun that we came to same conclussions:

https://github.com/cosmos/cosmos-sdk/blob/3c5dec40e3a22ba4c45abda549b8b300666fddca/x/bank/client/cli/cli_test.go#L193

Probably I will wait your merge to update mine, I needed to extract the SendTx in order to test the Auth module.